containers/airunner.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:22.04
+
+RUN apt-get update && apt-get install -y python3-pip git python-is-python3 less
+# All this stuff is needed to be able to render the airunner GUI (it already comes with Ubuntu-desktop usually!)
+RUN apt-get update && apt-get install -y '^libxcb.*-dev' libgl1 libsm6 libxrender1 libxext6 libglib2.0-dev libgl1-mesa-dev libxkbcommon-dev libdbus-1-3 libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev
+WORKDIR /root
+# To patch https://github.com/Capsize-Games/airunner/issues/16
+RUN pip install diffusers
```

Comment:
this is all handled correctly now for linux. There are two different repos - aihandler and now aihandlerwindows. I didn't update the installation instructions with the right info until today.